src/main.js

```diff
@@ -214,7 +214,14 @@ function readDiscoveryFeedProvidersInDirectory(directory) {
     let enumerator = null;
     let info = null;
     let providerBusDescriptors = [];
-    let languages = GLib.get_language_names();
+    let languages = [];
+    let contentLanguages = [];
+    let systemLanguages = GLib.get_language_names();
+
+    settings = new Gio.Settings({ schema_id: 'com.endlessm.DiscoveryFeed' });
+    contentLanguages = settings.get_strv('languages');
```

Comment:
You don't need to define the variable beforehand (also for `languages` below) -- you can just use `let contentLanguages = ...` here.

Reply:
nit: You can probably rename this key to something like `force-additional-languages`, since it forcibly includes more language other than those which would be available by default.

Reply:
Yes, thanks - renamed the key to `force-additional-languages`